frontend/src/components/map/index.js

```diff
@@ -1,28 +1,41 @@
-import React from "react";
-import { Map, TileLayer, Marker, Popup } from "react-leaflet";
-import styled from "styled-components";
+import React from 'react';
+import { Map as LeafletMap, TileLayer, Marker, Popup } from 'react-leaflet';
+import {geoLocation} from '../../utils/geolocation';
+import styled from 'styled-components';
 
-class MapComponent extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      lat: 51.505,
-      lng: -0.09,
-      zoom: 13
-    };
-  }
-
-  render() {
-    const position = [this.state.lat, this.state.lng];
-    return <div>Home</div>;
-  }
+const MapContainer = (props) => {
+    geoLocation()
```

Comment:
This should inside useEffect react hook: 

Code inside componentDidMount run once when the component is mounted. useEffect hook equivalent for this behavior is
```
useEffect(() => {
  getLocation().then(data => {
        // dispatch success action
      })
      .catch(error => {
         // dispatch error action
      });
}, []);
```